index.js

```diff
@@ -1,42 +1,58 @@
-/* eslint-disable no-unused-vars */
+/* eslint-disable import/named */
+import BookClass from './bookClass.js';
 /* eslint-disable no-use-before-define */
-const form = document.getElementById('form');
-const bookList = document.getElementById('bookList');
-const btn = document.getElementById('btn');
-let books = JSON.parse(localStorage.getItem('Atif')) || [];
+/* eslint-disable no-unused-vars */
+class Books {
+  constructor() {
+    this.bookList = document.getElementById('bookList');
+    this.bookTitle = document.getElementById('titleInput');
+    this.bookAuthor = document.getElementById('authorInput');
+    this.form = document.getElementById('form');
+    this.booksArray = [];
+    if (localStorage.getItem('Atif')) {
+      this.booksArray = JSON.parse(localStorage.getItem('Atif'));
+      this.displayBooks();
+    }
+  }
 
-function addBook(title, author) {
-  const book = { title, author };
-  books.push(book);
-  localStorage.setItem('Atif', JSON.stringify(books));
-  displayBooks();
-}
+  addBook = () => {
+    const title = this.bookTitle.value;
+    const author = this.bookAuthor.value;
+    const book = new BookClass(title, author);
+    this.booksArray.push(book);
+    this.displayBooks();
+    this.saveToLocalStorage();
+    this.bookAuthor.value = '';
+    this.bookTitle.value = '';
+  }
```

Comment:
here as well it's a good practice to implement single task method addBook is meant to create a new book object and add it to the list then return the updated list this way you can call displayBooks with the returned value 

`addBook = (title, author) =>{ this.booksArray.push({title,author}) return this.booksArray }`

Reply:
I have made the required change.